docs/config/template/index.template.html

```diff
@@ -11,6 +11,8 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic">
 <link rel="stylesheet" href="angular-material.min.css">
 <link rel="stylesheet" href="docs.css">
+  <script src="http://192.168.2.106/target/target-script-min.js#anonymous"></script>
```

Comment:
@ThomasBurleson @Frank3K Sorry, this line shouldn't be pushed to commit, I just introduced it for local debugging of an old device, which is not supporting touchAction and dev inspection.